ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/422__model_serving_llm.robot

```diff
@@ -136,6 +136,43 @@ Verify Model Pods Are Deleted When No Inference Service Is Present
     [Teardown]   Clean Up Test Project    test_ns=no-infer-kserve
     ...    isvc_names=${models_names}   isvc_delete=${FALSE}
 
+Verify User Can Autoscale Using Concurrency
+    [Tags]    ODS-XXX    WatsonX
+    [Setup]    Set Project And Runtime    namespace=autoscale-con
+    ${flan_model_name}=    Set Variable    flan-t5-small-caikit
+    ${model_name}=    Create List    ${flan_model_name}
+    Compile Inference Service YAML    isvc_name=${flan_model_name}
+    ...    sa_name=${DEFAULT_BUCKET_SA_NAME}
+    ...    model_storage_uri=s3://ods-ci-wisdom/flan-t5-small/
+    ...    auto_scale=True
+    Deploy Model Via CLI    isvc_filepath=${LLM_RESOURCES_DIRPATH}/caikit_isvc_filled.yaml
+    ...    namespace=autoscale-con
+    Wait For Pods To Be Ready    label_selector=serving.kserve.io/inferenceservice=${flan_model_name}
+    ...    namespace=autoscale-con
+    ${host}=    Get KServe Inference Host Via CLI    isvc_name=${flan_model_name}   namespace=autoscale-con
+    ${body}=    Set Variable    '{"text": "At what temperature does liquid Nitrogen boil?"}'
+    ${header}=    Set Variable    'mm-model-id: ${flan_model_name}'
+    ${cmd} =     Set Variable     grpcurl -d
+    ${cmd} =    Catenate   ${cmd} ${body}
+    ${cmd} =    Catenate   ${cmd} -H ${header}
+    ${cmd} =    Catenate   ${cmd} -insecure
+    ${cmd} =    Catenate   ${cmd}  ${host}:443
+    ${cmd} =    Catenate   ${cmd}  "caikit.runtime.Nlp.NlpService/TextGenerationTaskPredict"
+    FOR    ${index}    IN RANGE    30
+            ${rc}  ${response}=   	Run And Return Rc And Output    ${cmd}&
+            Run Keyword And Continue On Failure  Should Be Equal As Integers     ${rc}  ${0}
+    END
+    @{pod_lists}=    Oc Get    kind=Pod    namespace=autoscale-con
+        ...    label_selector=serving.kserve.io/inferenceservice=${flan_model_name}
+    ${count}=    Get Length   ${pod_lists}
+    IF   ${count} > ${1}
+         Log      Autoscale Using Concurrency is completed.Model Pod has been scaled up from 1 to $count
+    ELSE
+         FAIL     msg= Autoscale Using Concurrency has failed and Model pod has not been scaled up
```

Comment:
the keyword `Wait For Pods To Be Ready` from #885 has the feature to check replicas number

Reply:
hm that keyword would need to be slighlty changed to accept a generic check like > 1